Makefile

```diff
@@ -1,14 +1,21 @@
+GCLOUD_PROJECT:=$(shell gcloud config list project --format="value(core.project)")
 GRPCIO_VIRTUALENV=$(shell pwd)/grpc_python_venv
 PROTOC_CMD=$(GRPCIO_VIRTUALENV)/bin/python -m grpc.tools.protoc
 GOOGLEAPIS_PROTOS_DIR=$(shell pwd)/googleapis-pb
 
 help:
-	@echo 'Makefile for Shiny service                        '
-	@echo '                                                  '
-	@echo '   make generate    Generates the protobuf modules'
-	@echo '   make clean       Clean generated files         '
+	@echo 'Makefile for Shiny service                                     '
+	@echo '                                                               '
+	@echo '   make generate-proto-files    Generates the protobuf modules.'
+	@echo '   make docker-build    Builds the docker image.               '
+	@echo '   make docker-push    Pushes the docker image.                '
+	@echo '   make generate-container-engine-config    Generates the      '
+	@echo '       Container Engine deployment configuration.              '
+	@echo '   make deploy    Deploys the service to Container Engine.     '
+	@echo '   make clean    Clean generated files.                        '
 
-generate:
+.PHONY: generate-proto-files
+generate-proto-files:
```

Comment:
This isn't quite the right name, since the proto ~~files are~~ file is handwritten already. It's the `pb2` files that get generated.